Testscripts/Linux/nvme_blkdiscard.sh

```diff
@@ -46,6 +47,21 @@ Run_Blkdiscard() {
             LogMsg "Unmounted ${namespace}"
         fi
         #Run blkdiscard on partition
+        if [[ "$DISTRO_NAME" == "ubuntu" && "${DISTRO_VERSION}" == "14.04" ]]; then
```

Comment:
this is no longer required, we skip this test on 14.04 based on this merged PR:
https://github.com/LIS/LISAv2/pull/405/files